### PR TITLE
Respect typing array for Db

### PR DESCRIPTION
**Pull request by tolbon**

### CODE REVIEW

File: Model.php [hunk 1]
```diff
@@ -360,13 +360,13 @@ public function setSiteIds($idAlert, $idSites)
 
     public function deleteTriggeredAlertsForSite($idSite)
     {
-        $this->getDb()->query("DELETE FROM " . Common::prefixTable("alert_triggered") . " WHERE idsite = ?", $idSite);
+        $this->getDb()->query("DELETE FROM " . Common::prefixTable("alert_triggered") . " WHERE idsite = ?", array($idSite));
```
Comment (sgiehl) [inline Model.php]: personally I'd prefer to use the short array syntax `[]`

Comment (tolbon) [reply Model.php]: Yeah ok I change this. I prefer too.

Comment (tolbon) [reply Model.php]: @sgiehl Do you want i rename all array() to [] ?

Comment (sgiehl) [reply Model.php]: only if you change something in that line. Otherwise that would end up in too many changes and potential merge problems

**Comment by sgiehl** [comment]
Closing as source repo doesn't exist anymore and conflicts can't be resolved. Will add those changes manually

**Pull request closed without merging**